src/adapters/__tests__/pnFetch.test.ts

```diff
@@ -146,10 +150,8 @@ describe("getThirdPartyMessageDetails", () => {
       "x-pagopa-cx-taxid": aFiscalCode,
       signature:
         "sig1=:hNojB+wWw4A7SYF3qK1S01Y4UP5i2JZFYa2WOlMB4Np5iWmJSO0bDe2hrYRbcIWqVAFjuuCBRsB7lYQJkzbb6g==:",
-      "signature-input":
-        'sig1=("x-pagopa-lollipop-original-method" "x-pagopa-lollipop-original-url"); created=1618884475; keyid="test-key-rsa-pss"',
-      "x-pagopa-lollipop-assertion-ref":
-        "sha256-6LvipIvFuhyorHpUqK3HjySC5Y6gshXHFBhU9EJ4DoM=",
+      "signature-input": aSignatureInput,
```

Comment:
also, to reduce code duplication here we could create a constant with the object provided to this upper function `toHaveBeenCalledWith`, ready to be used across all the file.

Reply:
Fixed in [e919283](https://github.com/pagopa/io-backend/pull/1033/commits/e9192839d7cddfb5a992b8763defcb26ba16db65)